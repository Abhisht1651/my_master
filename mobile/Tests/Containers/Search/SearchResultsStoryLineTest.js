// @flow
import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import TestFile from '../../../App/Containers/Search/SearchResultsStoryLine'
import StoryLineShortesCard from '../../../App/Components/StoryLineShortesCard'

describe('Search Results StoryLine Screen', () => {
  it('should render the component itself correctly', () => {
    const wrapper = shallow(<TestFile />)
    expect(wrapper).toMatchSnapshot()
  })
  it('should render the StoryLineShortesCard Component', () => {
    const wrapper = shallow(<TestFile />)
    expect(wrapper.find(StoryLineShortesCard)).toHaveLength(9)
  })
})
