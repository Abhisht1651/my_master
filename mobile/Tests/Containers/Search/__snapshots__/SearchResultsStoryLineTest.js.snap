// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Results StoryLine Screen should render the component itself correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ComponentWrapper
    showsVerticalScrollIndicator={false}
  >
    <View
      style={
        Object {
          "marginVertical": 4,
          "width": 650,
        }
      }
    >
      <View
        key="0"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
      <View
        key="1"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
      <View
        key="2"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
      <View
        key="3"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
      <View
        key="4"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
      <View
        key="5"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
      <View
        key="6"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
      <View
        key="7"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
      <View
        key="8"
        style={
          Object {
            "marginTop": 20,
          }
        }
      >
        <storyLineShortestCard
          category="World"
          date="Sept 21, 2019"
          description="Ray Contreras talks about the different accents"
          source="Forbes"
          storyImage={
            Object {
              "default": "",
            }
          }
          storyTitle="UK exit from the European Union"
          trending={true}
          type={1}
        />
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
              "marginTop": 20,
              "width": 750,
            }
          }
        />
      </View>
    </View>
  </ComponentWrapper>
</View>
`;
